Specs/DateTime/Dutch/TimePeriodParser.json

```diff
@@ -805,20 +785,19 @@
           }
         },
         "Start": 16,
-        "Length": 17
+        "Length": 16
       }
     ]
   },
   {
-    "Input": "laten we elkaar van 13:00 tot 4 ontmoeten",
+    "Input": "laten we elkaar van 1 's middags tot 4 ontmoeten",
```

Comment:
Same comment as in the Extractor specs.

Reply:
added spec "laten we elkaar ontmoeten van 13:00 tot 4"
The original input is not supported because "4" is not at end of the string. 
The same happens in English.

Reply:
In such cases, you can leave the original spec in as well (in addition to any edited ones), mark it as NotSupported, and add a spec Comment describing the reason for the not supported.
This will help document expected behaviour and current limitations.

Reply:
ok, done